### ISSUE TITLE
Task: replace ListView → NavigationView

### ISSUE THREAD

**Issue by Jay-o-Way** [U-C-S/Hurl#168]
Willing to work on this :)
Idea: merge page **About** with page (general) **Settings**, for convenience.

References:
https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.navigationview?view=windows-app-sdk-1.6
https://learn.microsoft.com/windows/apps/design/controls/navigationview

**Comment by U-C-S** [comment]
Yes please

**Comment by Jay-o-Way** [comment]
ℹ️ Current status: issues with merging About + Settings, dealing with x:Bind to constants. For some reason it will work on About, but not when I copy to Settings. #TO-DO
```cs
    public string Version => Constants.VERSION;
    public string SourceCodeLink => Constants.SOURCE_CODE_LINK;
    public List<OSSLibrary> OpenSourceLibraries => [...]
```

**Comment by Jay-o-Way** [comment]
It's not that simple. Navigation is a complicated system, with classes and files. Best thing to do is create things (like files) from scratch and only copy/paste the essentials, but this means changes will become large quickly...

**Comment by U-C-S** [comment]
I played around this a bit back when I was implementing the existing system. I couldn't figure out how to use NavigationView from the docs and samples, looked into couple of other WinUI based OSS software, all of them had their own implementations upon listview, etc.

Anyway I will take a look, once again. If it doesn't work out, Have to just polish the existing ListView impl.

**Comment by Jay-o-Way** [comment]
Yeah, yesterday I started to try the long and large path, creating new blank pages et cetera. At least the first page _loads_, so that's a small win for now. I'll let you know how things progress...
